google-cloud-datastore/lib/google/cloud/datastore/version.rb

```diff
@@ -16,7 +16,7 @@
 module Google
   module Cloud
     module Datastore
-      VERSION = "1.5.0".freeze
+      VERSION = "1.5.1".freeze
```

Comment:
`1.6.0` since new functionality is introduced to the public API?

Reply:
I considered that. But ultimately felt like this wasn't adding new functionality as much as a new way to access existing functionality. So I felt that a patch release was appropriate.